internal/util/paramtable/component_param.go

```diff
@@ -75,6 +75,14 @@ func (p *ComponentParam) SetLogConfig(role string) {
 	p.BaseTable.SetLogConfig()
 }
 
+func (p *ComponentParam) RocksmqEnable() bool {
```

Comment:
for standalone we only support rmq.
So I guess rocksmqEnable is not a necessary config.

We need PulsarEnable/KafkaEnable in the distributed mode

Reply:
I think it would be better to offer more options.